### ISSUE TITLE
Fishnet 2

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#6]
Dataset: Fishnet
Data: 10%
Model: Best iNat21

Use a pre-trained swin-v2-base model pre-trained on iNat21 with the best validation loss, fine-tune it on Fishnet with 10% of the training data, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
